1-exercises/B-setting-values/exercise2.js

```diff
@@ -16,6 +16,7 @@ let student = {
 */
 
 // write code here
+student["attendance"] = 90;
```

Comment:
you can use dot notation when you know the name of the property you're accessing, i.e. `student.attendance`